haystack/document_store/faiss.py

```diff
@@ -398,14 +398,21 @@ def delete_all_documents(self, index: Optional[str] = None, filters: Optional[Di
 
     def delete_documents(self, index: Optional[str] = None, filters: Optional[Dict[str, List[str]]] = None):
         """
-        Delete all documents from the document store.
+        Delete documents from the document store. All documents are deleted if no filters are passed.
+
+        :param index: Index name to delete the document from.
+        :param filters: Optional filters to narrow down the documents to be deleted.
```

Comment:
Let's maybe add an example here as users might not be aware of what kind of `filters` they can pass. One example for deleting a document using a meta filter and one for `id` would be cool!

Reply:
Sure! I will add the same to all document stores then, I've seen that it lacks from all docstrings actually

Reply:
Now you got me thinking if this API supports filtering by `id` at all. Seems like `get_all_documents` looks into the `meta` field only, but I might be totally off-track... Going to verify this

Reply:
Yes, you are right. IDs are not supported yet - just meta fields. Let's not blow up the scope of this PR then. Can you please create a new issue for it? Then we can tackle it in the next sprint